charts/alertmanager/values.yaml

```
@@ -277,6 +281,13 @@ persistence:
     - ReadWriteOnce
   size: 50Mi
 
+  ## Storage emptyDir volume
+  ## Configure size limit and medium
+  ##
+  emptyDir:
+    medium: ""
+    sizeLimit: ""
```

imho this would be shorter and more flexible:
```suggestion
  emptyDir: {}
```